taccsite_cms/templates/guides/rsync_data_transfer.html

```diff
@@ -0,0 +1,510 @@
+      <p>
+        Assuming your TACC username is <code>jdoe</code> and you are affiliated with UT Austin, a <code>rsync</code> transfer that pushes
+        <code>my_file.txt</code>
+        from your local computer to the remote secure system would look like this:
+      </p>
+      <pre style="max-width: 1000px"><code><u>localhost$ </u>rsync ./my_file.txt jdoe@<kbd>host</kbd>:<kbd>/transfer/directory/path</kbd></code></pre>
+      <blockquote>
+        <p>If you have not done so already, enter this command in your terminal,
+          replacing the TACC username and your individualized transfer directory name appropriately.</p>
+      </blockquote>
+      <p>
+        If the command returns 0 in your terminal, the data transfer was successful.
+      </p>
+      <p>
+        If you wish to learn more about <code>rsync</code> and how to synchronize your file transfer, you can do so
+        <a target="_blank" href="https://man7.org/linux/man-pages/man1/rsync.1.html">the online <code>man</code> page for <code>rsync</code></a>
+        or follow the file transfer section of the user guide for the relevant storage system:
+      </p>
+      <ul>
+        <li>
+          <a target="_blank" href="https://portal.tacc.utexas.edu/user-guides/stampede2#transferring-rsync">Stampede2 User Guide: Transferring Files Using <code>rsync</code></a>
+        </li>
+        <li>
+          <a target="_blank" href="https://frontera-portal.tacc.utexas.edu/user-guide/files/#transferring-files-with-rsync">Frontera User Guide: Transferring Files with <code>rsync</code></a>
+        </li>
+        <li>
+          <a target="_blank" href="https://portal.tacc.utexas.edu/user-guides">TACC User Guides (for each system)</a>
+          <small>(select the relevant system for this portal, then search "<code>rsync</code>")</small>
+        </li>
+      </ul>
+
+      <h4 id="cyberduck_transfer">Transfer Data with Cyberduck</h4>
+
+      <p>
+        <a target="_blank" href="https://cyberduck.io/">Cyberduck</a> is a free graphical tool for data transfer and is an alternative to using the command line. With a drag-and-drop interface, it is easy to transfer a file from your local system to the remote secure system. You can use
+        <a target="_blank" href="https://cyberduck.io/">Cyberduck</a>
+        for Windows or macOS.
+      </p>
+      <h5 id="cy_windows">For Windows</h5>
+
+      <p>
+        <a target="_blank" href="https://cyberduck.io/download/">Download and install Cyberduck for Windows</a>
+        on your local machine.
+      </p>
+
+      <p>
+        Once installed, click “Open Connection” in the top left corner of your Cyberduck window.
+      </p>
+
+      <figure id="figure2">
+        <img src="https://a2cps-staging.tacc.utexas.edu/media/filer_public_thumbnails/filer_public/a9/9f/a99fa278-ae07-4b5e-9219-f9db3c2c89d5/picture2.png__1170x0_q85_subsampling-2_upscale.png" />
```

Comment:
From the generated pdf, it looks like these image links work, but I'm not entirely sure we want to be linking to one specific portal for these images.  I'm willing to be overruled if there isn't another way of handling this at this time.

Reply:
Known issue. I am using those for now to complete ticket within sprint. I have a "next up" ticket to **not** rely on external images for **all** pages: __Use Local Images for Guides Pages #147__